src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.cs

```diff
@@ -0,0 +1,574 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Thanks!  That helps.  Feel free to just note for future consideration how to cleanup the code that was copied and don't block on those comments.